src/renderer.ts

```diff
@@ -507,4 +507,5 @@ export {
     ImageElement,
     getHref,
     renderMedia,
+    renderCaption,
```

Comment:
Little bit cautious about exporting functions solely for use in the tests. Should we be testing the public API rather than a private one?